tools/run_tests/performance/templates/basic_template.yaml

```diff
@@ -0,0 +1,248 @@
+apiVersion: e2etest.grpc.io/v1
```

Comment:
the name "basic_template.yaml" is not very descriptive, expecially in the context where there's many other files and scripts in the tools/run_tests/performance/ directory.
How about something like "loadtest_template_all_languages.yaml"?

Reply:
Done. This template is actually "basic", in contrast to a template for tests that run with prebuilt images that is in the works. I have changed the name to "loadtest_template_basic_all_languages.yaml".